src/odemis/odemisd/test/yaml-merger-combination-include-extend-expected-result.yaml

```diff
@@ -0,0 +1,80 @@
+# Microscope file which uses multiple parents for a component:
+# * one parent (SEM Scan Interface) creates the component (CL PMT)
+# * one parent (CL Detector) uses the component
+
+SPARC2: {
+    class: Microscope,
+    role: sparc2,
+    children: ["SEM E-beam", "CL Detector"],
+}
+
+"SEM Scan Interface": {
+    class: semcomedi.SEMComedi,
+    role: null,
+    init: {
+           image: "simsem-fake-output.h5", # any large 16 bit image is fine
+           drift_period: 5, # seconds
+           username: "user1",
+           password: "complicated",
+    },
+    children: {
+       detector0: "CL PMT",
+       scanner: "SEM E-beam",
+    }
+}
+
+"CL Detector": {
+    class: pmtctrl.PMT,
+    role: cl-detector,
+    init: {
+        # Add via !inlcude
```

Comment:
I could add an include call here in the test cases but I don't know if that is needed